src/Icon.js

```diff
@@ -26,11 +29,15 @@ export default class Icon extends Component {
     const height = 20
     const width = 20
 
-    const { ratio, viewBox, icon, ...props } = this.props
+    const { className, ratio, viewBox, icon, ...props } = this.props
     const svg = icons[icon] && icons[icon].replace(/^<svg[^>]*>|<\/svg>$/g, '')
```

Comment:
This logic can be moved to after the `return <Spinner ...` logic to avoid checking if "spinner" exists in `icons` (we already know that it don't).